src/command_parser.c

```diff
@@ -71,15 +71,13 @@ static int _ModelRunCommand_ParseArgs(RedisModuleCtx *ctx, RedisModuleString **a
     }
     if ((*model)->inputs && (*model)->ninputs != ninputs) {
```

Comment:
Is the condition (*model)->inputs is still needed? The inputs and outputs arrays should be allocated in all backends now (not only in TF) right?